feature-styles/US_EnvironmentalManagementFacility.se

```diff
@@ -1,52 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<se:FeatureTypeStyle xmlns:us-emf="http://inspire.ec.europa.eu/schemas/us-emf/4.0" 
+<se:FeatureTypeStyle xmlns:us-emf="http://inspire.ec.europa.eu/schemas/us-emf/4.0"
 xmlns:act-core="http://inspire.ec.europa.eu/schemas/act-core/4.0"
-xmlns:ogc="http://www.opengis.net/ogc" xmlns:se="http://www.opengis.net/se" 
-xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink" 
+xmlns:ogc="http://www.opengis.net/ogc" xmlns:se="http://www.opengis.net/se"
+xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink"
 xsi:schemaLocation="http://www.opengis.net/se http://schemas.opengis.net/se/1.1.0/FeatureStyle.xsd"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1.0">
-	<se:Name>US.EnvironmentalManagementSite.Default</se:Name>
```

Comment:
Looking at chapter 11.2.2 of the DS, I think the old name was the correct one. But did not do a lot of research.

Reply:
In section 11.1 on page 121 it states the layer name is US.EnvironmentalManagementFacility.

Reply:
There are two style names US.EnvironmentalManagementInstallation.Default and US.EnvironmentalManagementSite.Default, but we can't offer both.

Reply:
Ok, I am never sure if FeatureTypeStyle.Name is providing the layer or the style name without proper testing ;)

Reply:
FeatureTypeStyle.Name is the style name. We cannot offer the two suggested styles in the specification because hc doesn't support that, so I added a neutral style name. What would you suggest?

Reply:
How about naming it something neutral and then adding two rules for the two cases filtering for the type?
Then they would at least look as expected.
If someone would want to fulfill the exact rules, they could still add custom SLDs then.

Reply:
Yes, we could do that. Add the two rules and an additional, default, catch-all rule.